crypto-ffi/bindings/js/CoreCrypto.ts

```diff
@@ -847,11 +866,13 @@ export class CoreCrypto {
         configuration: ConversationConfiguration = {}
     ) {
         try {
-            const {ciphersuite, externalSenders, custom = {}} = configuration || {};
+            const {ciphersuite, externalSenders, custom = {}, perDomainTrustAnchors} = configuration || {};
             const config = new CoreCrypto.#module.ConversationConfiguration(
                 ciphersuite,
                 externalSenders,
                 custom?.keyRotationSpan,
+                custom?.wirePolicy,
+                perDomainTrustAnchors as any[],
```

Comment:
@OtaK  is this the correct way of passing the parameter?

Reply:
Yes but keep in mind that you have set perDomainTrustAnchors as optional on the TS side and it is handled as non-optional on the rust side so you need to check for NULL/undefined on the rust side to make sure you don't get a crash when the user provides nothing. Alternatively, you acn make `perDomainTrustAnchors` have a default value on the TS side. Like so: `perDomainTrustAnchors = [];` on line 869